spec/systems/view_scraping_result_spec.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'views scraping result', type: :system do
+  include ActiveJob::TestHelper
+
+  context 'given completed scraping status' do
+    it 'displays google scraping result page' do
+      user = Fabricate(:user)
+      keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      VCR.use_cassette('with_top_position_adwords', record: :none) do
+        GoogleScrapingJob.perform_now(keyword.id, keyword.keyword)
```

Comment:
🤔

Reply:
Fixed in 114869b82f415ae12bd7f4a1bdf4ce13a4b23c28
and the reason same as [here](https://github.com/gutakk/ruby-google-scraper/pull/9#discussion_r509997414)